azure-pipelines.yml

```diff
@@ -17,7 +17,7 @@ variables:
   - group: sonarsource-build-variables
   # To make sure we don`t use MsBuild 17.
   - name: MsBuildPath
-    value: 'C:\Program Files\Microsoft Visual Studio\2022\Community\Msbuild\Current\Bin\msbuild.exe'
+    value: 'C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin\MSBuild.exe'
```

Comment:
Please leave a comment to revert the change. The best thing to do here is IMO what is described here: https://github.com/microsoft/vswhere/wiki/Find-MSBuild

Reply:
We will not revert this change. Please add an issue if you want to refactor the scripts and use Find-MSBuild.